services/web-server/package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "coverage": "c8 yarn test",
     "coverage:report": "c8 yarn test && c8 report --temp-directory ./coverage/tmp --reporter json --report-dir ../../artifacts",
-    "start": "NODE_ENV=development node src/main.js devServer",
+    "start": "cross-env NODE_ENV=development node src/main.js devServer",
```

Comment:
We will need to add `cross-env` with `yarn add -D cross-env -W` otherwise it won't be recognized here.

Reply:
I've run `yarn add -D cross-env -W`. It changed package.json and yarn.lock. But after commit merge conflict appeared in following place:
`"taskcluster-lib-scopes@link:../libraries/scopes":
  version "0.0.0"
  uid ""`

Reply:
Sometimes running `yarn` will help fix the conflict.